app/spdb/build.sbt

```diff
@@ -502,6 +503,30 @@ def gnodb(version: Version) = AppConfig(
   )
 ) extending List(odbproduction(version), gnodb_credentials(version))
 
+// GN AGS ODB: hbfauxodb-lv1 / gnauxodb
+def gnagsodb(version: Version) = AppConfig(
```

Comment:
Is there any risk on having an extra ODB, I guess it makes me nervous to have the actual dirs and servers on the props here. ODB may actively do something on those servers?